examples/simple-example/simple-example.pony

```diff
@@ -6,39 +6,37 @@ use "../../net_ssl"
 
 actor Main
   new create(env: Env) =>
-    let ssl = true
-    let limit = try
-      env.args(1)?.usize()?
-    else
-      1
-    end
+    let limit = try env.args(1)?.usize()? else 1 end
 
-    try
-      let auth = env.root as AmbientAuth
-      let sslctx: (SSLContext | None) = 
-        try
-          recover
-            // paths need to be adjusted to a absolute location or you need to run the example
-            // from a location where this relative path will be valid
-            SSLContext
-              .>set_authority(FilePath(auth, "../../examples/simple-example/cert.pem")?)?
-              .>set_cert(
-                FilePath(auth, "../../examples/simple-example/cert.pem")?,
-                FilePath(auth, "../../examples/simple-example/key.pem")?)?
-              .>set_client_verify(true)
-              .>set_server_verify(true)
-          end
+    let auth =
+      try env.root as AmbientAuth
```

Comment:
this isn't covered in the style guide but i think the try should be on its own line here.

i think we should make that clear in the style guide. if you have a multiline try, the the try should be on its own line.

Reply:
Do you mean that the `env.root as AmbientAuth` bit should be on a separate line after the `try` keyword?

Reply:
Yes.